### ISSUE TITLE
pkg-static segmentation fault during build-depends

### ISSUE THREAD

**Issue by 2bithacker** [freebsd/poudriere#268]
I'm having an issue with a port build via `poudriere bulk` that I can't seem to track down the cause of. When building Perl modules, the installation of perl5 into the build jail has a seg fault. I've tried doing a `pkg-static install` of the perl5 package outside of the jail and it works fine, but repeatably faults during the build.

```
===>   Installing existing package /packages/All/perl5-5.18.4_11.txz
[101amd64-default-job-01] Installing perl5-5.18.4_11...
[101amd64-default-job-01] Extracting perl5-5.18.4_11...Child process pid=30336 terminated abnormally: Segmentation fault
*** Error code 245
```

Full log at https://gist.github.com/2bithacker/15bfe6a57b2b40a78c03

**Comment by bapt** [comment]
This have been fixed in pkg 1.4.6

**Comment by 2bithacker** [comment]
Yup, that fixed it. Thanks!
